packages/stories/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@gitgraph/stories",
+  "version": "0.0.0",
```

Comment:
I removed it so lerna doesn't bother updating it. But it turns out package.json is not valid without, thus causing E2E tests to fail as it couldn't install `chromatic` anymore 🤷‍♂️